install.sh

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env bash
 set -e
 
-python setup.py install --force --user --prefix=
+which profanity > /dev/null
+which scanelf > /dev/null
+which tail > /dev/null
+which cut > /dev/null
+which tr > /dev/null
+which sed > /dev/null
+
+profanity="$(which profanity)"
+python_version="$(scanelf -n "${profanity}" | tail -n+2 | cut -d' ' -f2 | tr , '\n' | sed -nre 's/^libpython([0-9]+\.[0-9]+).*$/\1/p')"
```

Comment:
+1 for that. I'm not sure if this will work with BSD grep, so maybe someone could confirm if this works with macOS:

```bash
$ ldd $(which profanity)|grep -oP "^\slibpython\K(\d.\d)"
2.7
```